app/src/main/java/org/mozilla/tv/firefox/telemetry/TelemetryFactory.kt

```diff
@@ -14,6 +15,7 @@ import org.mozilla.telemetry.measurement.DefaultSearchMeasurement
 import org.mozilla.telemetry.net.TelemetryClient
 import org.mozilla.telemetry.ping.TelemetryCorePingBuilder
 import org.mozilla.telemetry.ping.TelemetryMobileEventPingBuilder
+import org.mozilla.telemetry.ping.TelemetryPocketEventPingBuilder
```

Comment:
The name of the ping needs to match the schema definition: https://github.com/mozilla-services/mozilla-pipeline-schemas/tree/dev/schemas/pocket/fire-tv-events

If it makes sense to have the same ping schema for multiple products then let's ask the team maintaining `mozilla-pipeline-schemas` (e.g. Frank) to rename that ping. It will prevent us from repeating the focus-events / mobile-events duplication :)